src/models/Settings.js

```diff
@@ -49,6 +49,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'stateToken': ['string', false, undefined],
       'username' : ['string', false],
       'signOutLink': ['string', false],
+      'cancelConsent': ['string', false],
```

Comment:
Suggestion: Rename to `consentCancelLink`?

Reply:
has to match backend name `cancelConsent` @larsjohansen-okta

Reply:
I changed it to ```consentCancelLink``` on the backend, so let's use that.

Reply:
ook